javascript/src/main/pages/AnalyzeMessageData/CountMessagesByUser.js

```diff
@@ -8,21 +8,25 @@ import aggregateUserMessageCount from "main/utils/aggregateUserMessageCount";
 const CountMessagesByUser = () => {
     const columns = [{
         dataField: 'name',
-        text: 'user'
+        text: 'User',
+        sort: true
```

Comment:
Hopefully quick change can you add the sort caret found on other pages for both users/messages? You can follow the code here 

https://github.com/ucsb-cs156-w21/proj-mapache-search/pull/139/files